src/main/java/com/dynxsty/dih4jda/interactions/commands/CommandRequirements.java

```diff
@@ -46,6 +57,15 @@ public final void requireRoles(Long... roles) {
 		requiredRoles = Arrays.stream(roles).collect(Collectors.toSet());
 	}
 
+	/**
+	 * Allows to require a set of {@link Long}s (role Ids) which are able to execute the corresponding command.
```

Comment:
```suggestion
	 * Defines a set of Role-IDs as {@link Long}s which are able to execute the corresponding command.
```
That could confuse some people as it sounds like all roles would be required to execute the corresponding command. Besides that, it doesn't allow to do something - it does it. When you use that method it's not optional to "allow" these roles